olive/passes/onnx/conversion.py

```diff
@@ -75,8 +75,10 @@ def _run_for_config(
 
         # TODO: add e2e test for model on cpu but data on gpu; model on gpu but data on cpu
         # put pytorch_model and dummy_inputs at the same device
-        pytorch_model.to("cpu")
-        dummy_inputs = tensor_data_to_device(dummy_inputs, "cpu")
+        # device = 'cpu' if self.accelerator_spec.accelerator_type == Device.CPU else 'cuda'
+        device = device_string_to_torch_device(self.accelerator_spec.accelerator_type)
+        pytorch_model.to(device)
```

Comment:
Can you put the device back on `cpu` once the export is done? the `pytorch_model` object persists in memory so we don't want it to keep using the `gpu` resources after this.

Reply:
Similar to https://github.com/microsoft/Olive/blob/main/olive/evaluator/olive_evaluator.py#L631

Reply:
Done!